locales/fr.json

```
@@ -245,24 +245,24 @@
     "log_letsencrypt_cert_install": "Installer le certificat Let's Encrypt sur le domaine '{}'",
     "log_selfsigned_cert_install": "Installer un certificat auto-signé sur le domaine '{}'",
     "log_letsencrypt_cert_renew": "Renouveler le certificat Let's Encrypt de '{}'",
-    "log_user_create": "Ajouter l'utilisateur '{}'",
-    "log_user_delete": "Supprimer l'utilisateur '{}'",
-    "log_user_update": "Mettre à jour les informations de l'utilisateur '{}'",
+    "log_user_create": "Ajouter le compte '{}'",
+    "log_user_delete": "Supprimer le compte '{}'",
+    "log_user_update": "Mettre à jour les informations du compte '{}'",
     "log_domain_main_domain": "Faire de '{}' le domaine principal",
     "log_tools_migrations_migrate_forward": "Exécuter les migrations",
     "log_tools_postinstall": "Faire la post-installation de votre serveur YunoHost",
     "log_tools_upgrade": "Mettre à jour les paquets du système",
     "log_tools_shutdown": "Éteindre votre serveur",
     "log_tools_reboot": "Redémarrer votre serveur",
-    "mail_unavailable": "Cette adresse email est réservée au groupe des administrateurs",
-    "good_practices_about_admin_password": "Vous êtes sur le point de définir un nouveau mot de passe administrateur. Le mot de passe doit comporter au moins 8 caractères, bien qu'il soit recommandé d'utiliser un mot de passe plus long (c'est-à-dire une phrase secrète) et/ou d'utiliser une combinaison de caractères (majuscules, minuscules, chiffres et caractères spéciaux).",
-    "good_practices_about_user_password": "Vous êtes sur le point de définir un nouveau mot de passe utilisateur. Le mot de passe doit comporter au moins 8 caractères, bien qu'il soit recommandé d'utiliser un mot de passe plus long (c'est-à-dire une phrase secrète) et/ou une combinaison de caractères (majuscules, minuscules, chiffres et caractères spéciaux).",
+    "mail_unavailable": "Cette adresse email est réservée au groupe des comptes administrateurs",
+    "good_practices_about_admin_password": "Vous êtes sur le point de définir un nouveau mot de passe de compte administrateur. Le mot de passe doit comporter au moins 8 caractères, bien qu'il soit recommandé d'utiliser un mot de passe plus long (c'est-à-dire une phrase secrète) et/ou d'utiliser une combinaison de caractères (majuscules, minuscules, chiffres et caractères spéciaux).",
```

Autre formulation possible, évitant l'adjectivation:
```suggestion
    "mail_unavailable": "Cette adresse email est réservée au groupe des comptes d'administration",
    "good_practices_about_admin_password": "Vous êtes sur le point de définir un nouveau mot de passe de compte d'administration. Le mot de passe doit comporter au moins 8 caractères, bien qu'il soit recommandé d'utiliser un mot de passe plus long (c'est-à-dire une phrase secrète) et/ou d'utiliser une combinaison de caractères (majuscules, minuscules, chiffres et caractères spéciaux).",
```

---

(erf c'était déjà utilisé plus bas ^^)

---

oui donc j'ai zappé de modifier ici ? on peut commit ?

---

ah non okay j'ai usé de 2 subterfuges différents
les deux sont good enough, plus qu'à décider le préféré

---

(pas compris si il faut faire un truc ou pas du coup)